packages/ember-toucan-form/src/-private/textarea-field.hbs

```diff
@@ -3,7 +3,8 @@
     @label={{@label}}
     @hint={{@hint}}
     @error={{this.mapErrors field.rawErrors}}
-    @value={{field.value}}
+    @value={{this.assertString field.value}}
```

Comment:
This is similar to https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/ember-headless-form/src/-private/components/field.ts#L256-L265. `field.value` could be anything, but `@value` only accepts strings. 

Had we types `USERDATA` as `Record<string, string>`, then we wouldn't have this issue. But it's just a generic `object`, as we want to allow arbitrary values. Like a checkbox group would be represented by an array...